openSUSE/open-build-service
# Eliminate caching of revisions and fetch them directly from the backend

This removes all the caching we did for revisions, replacing it with directly fetching revisions from the backend. Despite the lack of caching, this is faster, since we always fetch just the revisions we need while we need them.

To test this:
1. Visit the package page and check if the revision element is correct
2. Visit the package revisions page and check if the revisions are correct

Fixes #13866
